### PR TITLE
First pass at jobs gui

### PR DESCRIPTION
**Pull request by brandonbloom**
Joint work with @kendru - should be safe for main branch, but needs lots of follow-up work.

### CODE REVIEW

**Comment by brandonbloom** [comment]
@jwmza look this over please - then let's chat soon about what we want to do with this

**Comment by jwmza** [comment]
Will do

**Comment by jwmza** [comment]
Well, this will definitely require adapting to the new layout and table component changes.

Even in the current version:

- The pages need a white background, and probably some padding too. I've created a new `MonoPanel` base layout for generic pages in the branch `left-nav-layout-overhaul`.
- Probably a good idea to use the `CheckeredTableWrapper` component around the tables here and use `tbody` in addition to `thead` to segment the table DOM tree properly and activate the generic checkered table styling.

I'm not sure how to test the actual jobs feature yet, will need some filling in on that @brandonbloom

File: gui/src/components/Link.svelte [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+  import * as router from 'svelte-spa-router';
+
+  export let href: string;
+</script>
+
+<a
```
Comment (jwmza) [inline gui/src/components/Link.svelte:7]: Looks good. Might need an option to `display: block` as we use this broadly. Not sure yet.

File: gui/src/pages/Job.svelte [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  export let params = { job: '' };
+
+  const jobId = params.job;
+
+  const tasks = api.kernel.describeTasks({ jobIds: [jobId] });
+</script>
+
+<Layout showBackButton backButtonRoute="#/jobs">
+  <!-- TODO: common loader component -->
```
Comment (jwmza) [inline gui/src/pages/Job.svelte:14]: Maybe wrap everything within `Layout` here inside a `<section>...</section>` that has `min-height: 100vh;` and `background: #ffffff` or wait until #131 lands and use `<MonoPanel>...</MonoPanel>`

File: gui/src/pages/Jobs.svelte [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  <section>
+    Jobs
+    <!-- TODO: common loader component -->
+    {#await jobs}
+      Loading...
+    {:then jobs}
+      {#if jobs.length === 0}
+        <i>None</i>
+      {:else}
+        <table>
```
Comment (jwmza) [inline gui/src/pages/Jobs.svelte:22]: Suggestion: Wrap in `<CheckeredTableWrapper>` and use `<tbody>` in addition to `<thead>`

**Review by jwmza — COMMENTED** [review]
LGTM, But I haven't tested the Tree stuff yet.

File: gui/src/pages/Job.svelte (hunk 2 above)
Comment (brandonbloom) [reply gui/src/pages/Job.svelte:14]: Going to wait for #131 - this UI is expectedly ugly as sin still.

**Comment by brandonbloom** [comment]
merged in to the destroy-and-apply-progress branch

**Pull request merged**